### ISSUE TITLE
Fails when an item is in stock on currys

### ISSUE THREAD

**Issue by Alienwarez123** [Prince25/StockAlertBot#65]
When I get a stock alert this error occurs
`
10:59:20 AM: ***** In Stock at Currys *****:  Tornado R5X Gaming PC - AMD Ryzen 5, RTX 3060 Ti, 2 TB HDD &amp; 512 GB SSD
https://www.currys.co.uk/gbuk/gaming/pc-gaming/gaming-pcs/pc-specialist-tornado-r5x-gaming-pc-amd-ryzen-5-rtx-3060-ti-2-tb-hdd-512-gb-ssd-10219569-pdt.html
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: spawn xdg-open ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn xdg-open',
  path: 'xdg-open',
  spawnargs: [
    'https://www.currys.co.uk/gbuk/gaming/pc-gaming/gaming-pcs/pc-specialist-tornado-r5x-gaming-pc-amd-ryzen-5-rtx-3060-ti-2-tb-hdd-512-gb-ssd-10219569-pdt.html'
  ]
}`

**Comment by Prince25** [comment]
This error is peculiar because it isn't caused by my code. Did you run `npm i`? If so, can you tell me your node version by running `node -v`? What notifications do you have turned on? As you can see, it's working well for me:
![image](https://user-images.githubusercontent.com/28243638/113219093-91161e80-9235-11eb-8110-26f4a014d92b.png)

**Comment by Alienwarez123** [comment]
My node version is v14.16.0, I am running Ubuntu 18.04.5 LTS x86_64, and I'm going to take a guess now if its about the webhook notifications that could be it but it only fails on linux since I can run the exact same thing fine on windows. I modified the utils/notification/webhook.js and added some small support for IFTTT.

![image](https://user-images.githubusercontent.com/42479938/113230378-ca2fac80-9290-11eb-8ca8-de8d23d96def.png)
^^^ This is it running on windows perfectly fine with the modifications

![image](https://user-images.githubusercontent.com/42479938/113232998-247f3c00-9296-11eb-9474-62886f114383.png)
^^^ This is it running on a free hosting service with the exact same files as windows 

![image](https://user-images.githubusercontent.com/42479938/113230477-f814f100-9290-11eb-90b3-2fb3b95864c7.png)
^^^ This is it failing on the ubuntu machine with the exact same files as windows
UPDATE:
Well by just updating it from doing another git clone then running npm install then updating the config.json and the webhook.js with the modified details, it seems to work perfectly now so idk if its something to do with your new commits or just some corrupted file somewhere but I don't get how that will happen since I have used the same zip file across multiple machines that worked and had uploaded this zip file many times yesterday to that machine and it failed, but thank you for your help in trying to fix the issue, I'm not too familiar with nodejs so this could be my fault trying to attempting to modify it out of its intended function but I understand how the program basically functions from previous knowledge on other languages.

https://pastebin.com/m4RVvgCc webhook.js then on the config.json I just added ifttt to the array of supported domains

PS This application you made is brilliant, I'm hoping this will help me get a 30 series card for my PC.
